apis/managed-dimensions/lib/shapes/managed-dimension.ts

```diff
@@ -0,0 +1,91 @@
+import { dash, qudt, rdf, schema, sh, time, xsd } from '@tpluscode/rdf-ns-builders'
+import { supportedLanguages } from '@cube-creator/core/languages'
+import type { Initializer } from '@tpluscode/rdfine/RdfResource'
+import type { NodeShape } from '@rdfine/shacl'
+import $rdf from 'rdf-ext'
+import { meta, sh1 } from '@cube-creator/core/namespace'
+
+export default (): Initializer<NodeShape> => ({
+  closed: true,
+  property: [{
+    name: 'Name',
+    path: schema.name,
+    uniqueLang: true,
+    minCount: 1,
+    maxCount: 4,
+    languageIn: supportedLanguages,
+    order: 10,
+  }, {
+    name: 'Valid from',
+    path: schema.validFrom,
+    maxCount: 1,
+    datatype: xsd.dateTime,
+    defaultValue: $rdf.literal(new Date().toISOString(), xsd.dateTime),
+    order: 20,
+  }, {
+    path: sh.property,
```

Comment:
Any way we can avoid a nested form here? If not, I would add something like `name: ' '` to force it not to display `sh:property` as label.

Reply:
I actually added a label/description which this deserves.

Cannot avoid nesting unless we try to support complex `sh:path`